### DISCUSSION TITLE
How do I remove the menu button (3 dots) from the navigation bar?

### DISCUSSION THREAD

**Discussion by Virock** [unoplatform/uno.toolkit.ui#989] [Q&A]
Hello community. I need help removing the menu button in the navigation bar.

I have this in my xaml file: `<utu:NavigationBar Content="Login/ Sign Up" />`
But it's rendered like this:
![1](https://github.com/unoplatform/uno/assets/31706335/f0398e98-2155-4b53-923d-503ad1e5fd5b)
How do I remove this:
![2](https://github.com/unoplatform/uno/assets/31706335/e46fac54-fef2-409e-9ad3-224e70602977)
I have tried setting the OverflowButtonVisibility to collapsed as such:
`<utu:NavigationBar Content="Login/ Sign Up" OverflowButtonVisibility="Collapsed" />` but it doesn't have any effect.

How do I remove the menu button?

Edit: The menu button does not appear if the current page is the root page. It only appears after navigating to a page which causes the Navigation stack to have more than one item.

**Comment by kazo0** [comment]
Hi @Virock, sorry I had missed this question! This is very odd indeed, the ellipsis button should definitely not be present in this case. It should usually only be there if you have SecondaryCommands defined or PrimaryCommands that don't fit with the current screen width which doesn't seem to be the case.

I will log this as a bug and investigate, thanks!

**Reply by Virock** [reply]
Alright. Thank you.
